### PR TITLE
Add some post deployment routing docs

### PR DESCRIPTION
**Pull request by rjw1**

### CODE REVIEW

**Comment by rgarner** [comment]
Have rebased this, not sure whether it's still current, @rjw1? If so please click merge at your convenience

File: CF/routing-readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Routing
+
+Once the cdn service is set up and the apps have been deployed we need to set up
+some final routing.
+
+```
+cf target -s staging
+cf map-route ccs-rmi-app-staging --hostname www staging.rmi-paas.dxw.net
```
Comment (tahb) [inline CF/routing-readme.md]: When running this for sandbox we found that the `bind-route-service` command will fail with `api not found` unless the route is mapped before hand:

```
cf map-route ccs-rmi-api-staging --hostname api staging.rmi-paas.dxw.net
```

Same for production further down.

File: CF/routing-readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Routing
+
+Once the cdn service is set up and the apps have been deployed we need to set up
+some final routing.
+
+```
+cf target -s staging
```
Comment (tahb) [inline CF/routing-readme.md]:
```suggestion
cf target -s staging
cf map-route ccs-rmi-api-staging --hostname api staging.rmi-paas.dxw.net
```

File: CF/routing-readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+cf map-route ccs-rmi-app-staging --hostname www staging.rmi-paas.dxw.net
+cf bind-route-service staging.rmi-paas.dxw.net ccs-rmi-api-admin-route-staging --hostname api
+cf target -s prod
```
Comment (tahb) [inline CF/routing-readme.md]:
```suggestion
cf target -s prod
cf map-route ccs-rmi-api-prod --hostname api prod.rmi-paas.dxw.net
```

**Pull request merged**